docs/_source/getting_started/installation/configurations/user_management.md

```diff
@@ -11,15 +11,15 @@ Let's first describe Argilla's user management model:
 A Argilla user is defined by the following fields:
 
 - `username`: The username to use for login into the Webapp.
-- `email`(optional): The user's email.
-- `fullname` (optional): The user's full name
-- `disabled`(optional): Whether this use is enabled (and can interact with Argilla), this might be useful for disabling user access temporarily.
-- `workspaces`(optional): The team workspaces where the user has read and write access (both from the Webapp and the Python client). If this field is not defined the user will be a super-user and have access to all datasets in the instance. If this field is set to an empty list `[]` the user will only have access to her user workspace. Read more about workspaces and users below.
+- `first_name`: The user's first name
+- `last_name` (optional): The user's last name
+- `role`: The user's role in Argilla. Available roles are: "admin" and "annotator". Only "admin" users can create and delete workspaces and datasets
```

Comment:
Can annotators add labels?

Reply:
nope

Reply:
We should mention it

Reply:
Add full-stops at the end of the lines. The other points have them

Reply:
@dvsrepo the full role management is not implemented on this release. We're giving a similar behavior to the current one by explicit role assignment.

The role permissions should be reviewed in future versions.